test.properties

```diff
@@ -9574,7 +9585,8 @@
         Plugin Installer
 
     testray.team.object.component.names=\
-        Object
+        Object,\
+        Upgrades Objects
```

Comment:
I think this should be `Upgrades Object` to follow our naming convention. I'm not entirely opposed to renaming `Object` to `Objects` but that would need a separate ticket and announcement to our team so that everyone is on the same page.

Reply:
Not sure why the rest of the review comments are in a separate section.